sql/promscale--0.0.0.sql

```diff
@@ -356,28 +362,6 @@ END
 $block$
 ;
 
```

Comment:
why is it safe to delete this code for tracing hypertables?

Reply:
We want pg_dump to manage the dump/restore process for the chunks.

Reply:
if we make them config tables, then our `create extension` code has to `create table` the chunks, but since chunks are created dynamically, and I don't think the timescaledb schemas are loaded yet, we don't have enough information to know what chunks we need to create. Have to leave it to pg_dump to do.

Reply:
I'm not sure it answers Mat's question. My understanding is that the same code appears in `incremental/015-tracing-compression.sql`, so we don't need it in `0.0.0`